src/components/Intranet/Categories/AddCategoryModal.tsx

```
@@ -0,0 +1,157 @@
+    isOpen,
+    onOpenChange,
+    onSuccess,
+}: AddCategoryModalProps) {
+    const { t } = useTranslation();
+    const [file, setFile] = useState<File | null>(null);
+    const [categoryName, setCategoryName] = useState("");
+    const [previewImage, setPreviewImage] = useState<string | null>(null);
+    const fileInputRef = useRef<HTMLInputElement>(null);
+
+    const handleFileChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        const selectedFile = e.target.files && e.target.files[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)